applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -559,17 +609,20 @@ int ble_mcp_conn_disconnected(struct bt_conn *conn)
 	return 0;
 }
 
-#if (CONFIG_BT_VCP_VOL_CTLR)
 int ble_vcs_client_init(void)
 {
-	static struct bt_vcp_vol_ctlr_cb vcs_client_callback;
+	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+		static struct bt_vcp_vol_ctlr_cb vcs_client_callback;
+
+		vcs_client_callback.discover = vcs_discover_cb_handler;
+		vcs_client_callback.state = vcs_state_ctlr_cb_handler;
+		vcs_client_callback.flags = vcs_flags_ctlr_cb_handler;
+		return bt_vcp_vol_ctlr_cb_register(&vcs_client_callback);
+	}
```

Comment:
To reduce indentations:
```suggestion
	if (!IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
	    LOG_ERR("VCP volume controller not enabled");
	    return -ECANCELED;
        }
		
	static struct bt_vcp_vol_ctlr_cb vcs_client_callback;

	vcs_client_callback.discover = vcs_discover_cb_handler;
	vcs_client_callback.state = vcs_state_ctlr_cb_handler;
	vcs_client_callback.flags = vcs_flags_ctlr_cb_handler;
	return bt_vcp_vol_ctlr_cb_register(&vcs_client_callback);
	}
```

Reply:
Same below

Reply:
Changed the structure for all the instances